simulation/network_simulator.cc

```diff
@@ -27,10 +28,10 @@ int main(int argc, char* argv[])
 
     CoE::EsiParser parser;
     auto& coe_dict = CoE::dictionary();
-    coe_dict = parser.load("ingenia_esi.xml");
+    coe_dict = parser.load("evs-net-01_esi_2.2.3.xml");
```

Comment:
Is this file accessible somewhere ?

Reply:
No, but it can be downloaded on this manufacturer website :)

On a side note: the whole mechanism shall be modified to better described all the slaves that run on the simulator (who mentioned an ENI file?)